.gitignore

```
@@ -31,3 +31,4 @@ test-results.xml
 # Getting started / demo instructions
 devnet/
 testnet/
+docs/yarn.lock
```

```suggestion
testnet/
```

We should keep the `docs/yarn.lock` in the git index.